study.md

```diff
@@ -31,7 +31,7 @@ In your own words, define what the responsibilities of the model layer are in
 Rails.
 
 ```md
-<!-- your answer here -->
+The responsibility for the model layer is to chat to the backend. It talks to the database and also conducts backend logic (server side).
```

Comment:
The model layer is part of the backend, so it's responsibility cannot be to "chat" with the backend. Which parts of the backend does the model interact with, and how?